doc/Building.md

```diff
@@ -2,133 +2,108 @@
 The `package.json` file in the main folder includes the following scripts:
 
-
 ### `npm run typescript:build`
 
 Compiles mediasoup TypeScript code (`lib` folder) JavaScript and places it into the `lib` directory.
 
-
 ### `npm run typescript:watch`
 
 Compiles mediasoup TypeScript code (`lib` folder) JavaScript, places it into the `lib` directory an watches for changes in the TypeScript files.
 
-
 ### `npm run worker:build`
 
 Builds the `mediasoup-worker` binary. It invokes `make`below.
 
-
 ### `npm run lint`
 
 Runs both `npm run lint:node` and `npm run lint:worker`.
 
-
 ### `npm run lint:node`
 
 Validates mediasoup JavaScript files using [ESLint](https://eslint.org).
 
-
 ### `npm run lint:worker`
 
 Validates mediasoup-worker C++ files using [clang-format](https://clang.llvm.org/docs/ClangFormat.html). It invokes `make lint` below.
 
-
 ### `npm run format:worker`
 
 Rewrites mediasoup-worker C++ files using [clang-format](https://clang.llvm.org/docs/ClangFormat.html). It invokes `make format` below.
 
-
 ### `npm run test`
 
 Runs both `npm run test:node` and `npm run test:worker`.
 
-
 ### `npm run test:node`
 
 Runs [Jest](https://jestjs.io) test units located at `test/` folder.
 
-
 ### `npm run test:worker`
 
 Runs [Catch2](https://github.com/catchorg/Catch2) test units located at `worker/test/` folder. It invokes `make test` below.
 
-
 ### `npm run coverage:node`
 
 Same as `npm run test:node` but it also opens a browser window with JavaScript coverage results.
 
-
 ### `npm run install-deps:node`
 
 Installs NPM dependencies and updates `package-lock.json`.
 
-
 ### `npm run install-clang-tools`
 
 Installs clang tools needed for local development.
 
-
 ## Rust
 
 The only special feature in Rust case is special environment variable `KEEP_BUILD_ARTIFACTS`, that when set to `1` will allow incremental recompilation of changed C++ sources during hacking on mediasoup.
 It is not necessary for normal usage of mediasoup as a dependency.
 
-
 ## Makefile
 
 The `worker` folder contains a `Makefile` for the mediasoup-worker C++ subproject. It includes the following tasks:
 
-
 ### `make` or `make mediasoup-worker`
 
 Alias of ``make mediasoup-worker` below.
 
-
 ### `make meson-ninja`
 
 Installs `meson` and `ninja`.
 
-
 ### `make clean`
 
 Cleans built objects and binaries.
 
-
 ### `make clean-build`
 
 Cleans built objects and other artifacts, but keeps `mediasoup-worker` binary in place.
 
-
 ### `make clean-pip`
 
 Cleans `meson` and `ninja` installed in local prefix with pip.
 
-
 ### `make clean-subprojects`
 
 Cleans subprojects downloaded with Meson.
 
-
 ### `make clean-all`
 
 Cleans built objects and binaries, `meson` and `ninja` installed in local prefix with pip and all subprojects downloaded with Meson.
 
-
 ### `make update-wrap-file`
 
 Update the wrap file of a subproject with Meson. Usage example:
 
 ```bash
-$ cd worker
-$ make update-wrap-file SUBPROJECT=openssl
+cd worker
+make update-wrap-file SUBPROJECT=openssl
```

Comment:
Same here. Why are you removing the `$` symbol? It's cosmetic and super opinionated, but we put it on purpose.

Reply:
Found the rule: https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md014.md

> Rationale: It is easier to copy/paste and less noisy if the dollar signs are omitted when they are not needed. See https://cirosantilli.com/markdown-style-guide#dollar-signs-in-shell-code for more information.

If you still wants them, I can revert them back, and maybe add a `.markdownlintrc` file to prevent happening again, the same for the two empty lines if you want.